### PR TITLE
Run E2E tests in parallel

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Made E2E tests run in parallel

## Why?

Some tests have to sleep for ~10 etc seconds. Running in parallel speeds up the total execution time 

## Checklist

1. Closes 

2. How was this tested:

CI

3. Any docs updates needed?

### CODE REVIEW

File: app/server_test.go [hunk 1]
```diff
@@ -22,40 +22,30 @@
...
-
-	// Wait for the app to start
-	time.Sleep(time.Second * 2)
-
-	cmd.Process.Signal(syscall.SIGTERM)
-	_ = cmd.Wait()
-
-	assert.Equal(t, 0, cmd.ProcessState.ExitCode())
-}
+// func TestServerInterruptRC(t *testing.T) {
```
Comment (bergundy) [inline app/server_test.go:25]: Intentional?

File: tests/namespace_test.go
Comment (bergundy) [inline tests/namespace_test.go:1]: Doesn't seem to belong to this PR

File: app/server_test.go (hunk 1 above)
Comment (feedmeapples) [reply app/server_test.go:25]: just pushed everything, cleaning things up and will rebase Namespace Update on top of this

**Review by samanbarghi — APPROVED** [review]
